src/main/java/com/dtolabs/rundeck/plugin/overthere/OTWinRMNodeExecutor.java

```diff
@@ -89,6 +90,11 @@
     public static final String HOSTNAME_TRUST_ALL = "all";
 
 
+    public static final String CONFIG_CMD_CMD = "CMD";
```

Comment:
Applying the naming observed on other parameters ("WINRM_HOSTNAME_TRUST" then "HOSTNAME_TRUST_..."), I would suggest going instead for "CMD_TYPE_CMD"